app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -272,6 +272,23 @@ def convert_memory_to_bytes(memory)
       convert_memory(memory, :b)
     end
 
+    def console(uuid)
+      vm = find_vm_by_uuid(uuid)
+      vnc_details = client.vminstances.get_vnc_console_details(vm.name, namespace)
+      token = Base64.encode64(vnc_details[:token]).delete!("\n").delete("==")
+      token_protocol =  "base64url.bearer.authorization.k8s.io.#{token}"
```

Comment:
there is an extra space after the =
also, "base64url.bearer.authorization.k8s.io." can be pulled into a constant